File: src/main/java/ru/vaschenko/ParkPoint/dto/request/SubscriptionRequestDto.java
package ru.vaschenko.ParkPoint.dto.request;

import java.time.LocalTime;
import java.util.List;

public record SubscriptionRequestDto(
        Long parkingSpaceId,
        Long clientId,
        List<Integer> dayOfWeak,
        LocalTime startTime,
        LocalTime endTime
) {}

File: src/dto/SubscriptionRequestDto.js
export class SubscriptionRequestDto {
    constructor(parkingSpaceId, clientId, dayOfWeek, startTime, endTime) {
      this.parkingSpaceId = parkingSpaceId;
      this.clientId = clientId;
      this.dayOfWeek = dayOfWeek;
      this.startTime = startTime;
      this.endTime = endTime;
    }
  }
  